ui/modal/create_watch_only_modal.go

```diff
@@ -107,6 +107,8 @@ func (cm *CreateWatchOnlyModal) WatchOnlyCreated(callback func(walletName, extPu
 }
 
 func (cm *CreateWatchOnlyModal) Handle() {
+	cm.modal.SetDisabled(cm.isLoading)
```

Comment:
`cm.isLoading` might be false at this point and the modal will be enabled but could change to true later in this method in line 139 and the modal will not be disabled. Also worth noting that the `cm.SetLoading(false)` method may be called outside this file and the modal will need to be enabled if previously disabled (as it should be disabled when cm.isLoading became true).

In general, the modal's enabled state should be changed when some action makes it necessary to change the modal state such as when a button is clicked and the form is being submitted or when the form submission completes and the modal can be renabled. In the previous 2 files I've reviewed, there's a `SetLoading` method that seems to be designed to handle moments when the modal is about to be busy or stops being busy. That would be the ideal place to do `modal.SetDisabled`.